Mathlib/Order/CompleteLattice.lean

```diff
@@ -569,6 +569,12 @@ theorem sSup_insert {a : α} {s : Set α} : sSup (insert a s) = a ⊔ sSup s :=
   ((isLUB_sSup s).insert a).sSup_eq
 #align Sup_insert sSup_insert
 
+theorem sup_sSup {s : Set α} {a : α} :
+    a ⊔ sSup s = sSup (s ∪ {a}) := by simp only [Set.union_singleton, sSup_insert]
```

Comment:
I'm not sure that we need these lemmas. E.g., RHS is not in the `simp`-normal form.

Reply:
Indeed, they were not used (in sphere-eversion either). Deleted.